applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_process.py

```diff
@@ -60,6 +62,11 @@ def __init__(self, Model, settings ):
         self.u_inf = self.free_stream_mach * self.free_stream_speed_of_sound
         self.free_stream_velocity = KratosMultiphysics.Vector(3)
 
+        if self.angle_of_attack_units == "radians":
+            KratosMultiphysics.Logger.PrintWarning("ApplyFarFieldProcess", " Using 'radians' as default angle of attack unit.")
```

Comment:
```suggestion
            KratosMultiphysics.Logger.PrintWarning("ApplyFarFieldProcess", " Using 'radians' as angle of attack unit. This will be deprecated soon.")
```